content/en/docs/topics/styles/index.md

```diff
@@ -610,6 +611,24 @@ since the rule used `scope: raw`.
 
 See [Scoping](/docs/topics/scoping) for more information.
 
+## Actions
+
+Actions in rules indicate a way for external tools with Vale integration to provide methods for correcting style issues. For example with code actions in VSCode.
+
+The Vale CLI tool doesn't directly do anything with the value of this field and it's up to the external tool to support any actions.
+
+While styles can use whatever value they want for actions and tools can implement the actions how they want, there are a series of standard actions that existing rules and tools use.
+
+{{< details "Key summary" >}}
+| Name | Parameters | Suggested implementation |
+| :--- | :--- | :--- |
+| `replace` | Values from the `swap` key | Swap matched value for suggestion in `swap` |
+| `remove` | None | Remove the instance of the matched token. |
+| `suggest` |  |  |
```

Comment:
@jdkato I am a little unsure if these last three are "needed" any more? As much as any are needed, but I feel like the other actions have the same function?